app/service/cdo.js

```diff
@@ -23,14 +23,24 @@ CdoService.prototype.getTaskList = async function (cdoId) {
   return this.cdoRepository.getCdoTaskList(cdoId)
 }
 
-CdoService.prototype.sendApplicationPack = async function (cdoId, user) {
+CdoService.prototype.sendApplicationPack = async function (cdoId, sentDate, user) {
   const cdoTaskList = await this.cdoRepository.getCdoTaskList(cdoId)
-  const sendEvent = async (preChanged, postChanged) => {
-    await sendUpdateToAudit(EXEMPTION, preChanged, postChanged, user)
+  const activityType = await getActivityByLabel('Application pack')
```

Comment:
It probably makes sense to put 'Application pack' as a constant in say events.js and then refer to it from there in the few places it's needed

Reply:
Great idea, done.